src/components/download-links/index.js

```diff
@@ -23,6 +23,13 @@ export default () => {
 
   subscribe('downloads:loaded', (data) => {
     Fetch(data)
+    .then(data => {
+        // disable downloads button when no links available
+        if (!data || !data.length)
+            $container.querySelector("svg").setAttribute("disabled", true);
+
+        return data;
+    })
```

Comment:
Can we just hide the button instead? atm it's effectively disabled already, since nothing happens when you click it.

Reply:
That's fair, prob better actually

Reply:
Hiding per 35a7658.